lib/rails/generators/appoptics_apm/templates/appoptics_apm_initializer.rb

```diff
@@ -128,14 +128,16 @@
   # Common Ruby and Rails loggers are auto-instrumented, so that they can include
   # the current Trace ID in log messages.
   #
-  # The added string will look like: "ao.traceId=7435A9FE510AE4533414D425DADF4E180D2B4E36-0"
+  # The added string will look like:
+  # "trace_id=7435a9fe510ae4533414d425dadf4e18 span_id=49e60702469db05f trace_flags=00"
```

Comment:
💯 Thanks for the clear 3-line spec of "Trace Context in Logs" . Been looking for something like that....

Reply:
:)